BP/common/religion/religions/00_baltic.txt

```diff
@@ -194,6 +194,7 @@
 	}
 
 	faiths = {
+		#WtWSMS Differences: No raiding, **can** declare HW
```

Comment:
I'm not convinced on these differences.

Reply:
Raiding is tied unreformed pagan, tribal government, or certain dynasty modifiers (not something implicit in a religion) and HW casus belli requires organization and non-pacficism. So a reformed Baltic won't be able to raid, but will be able to HW.